components/gateway/internal/auditlog/service.go

```diff
@@ -7,38 +7,25 @@ import (
 	"strings"
 	"time"
 
+	"github.com/kyma-incubator/compass/components/director/pkg/correlation"
 	"github.com/kyma-incubator/compass/components/gateway/pkg/auditlog/model"
 	"github.com/kyma-incubator/compass/components/gateway/pkg/proxy"
 	"github.com/pkg/errors"
 )
 
-type Message struct {
```

Comment:
We've decided to move it up in the proxy package due to circular a dependency problem. We couldn't instantiate an auditlog.Message instance because it carries in itself a proxy.Claims instance.